doc/abel.rst

```diff
@@ -169,3 +177,13 @@ abel.benchmark module
     :members:
     :undoc-members:
     :show-inheritance:
+
+abel.tests module
```

Comment:
I think, @rth has removed this in 5636c402c3791424859335f8d78ddb41ac47e5e9. I don't know how it got back. Need to remove again?

Reply:
Yes, there is no point in including tests as part of the public API in the docs.

Reply:
There is also `test_suite="abel.tests.run_cli"` remaining [in setup.py](https://github.com/PyAbel/PyAbel/blob/be2d1adb1b20a664ac6f757d1c40fb737222ba63/setup.py#L116). Perhaps, should be removed or updated?

Reply:
OK, removed this and `test_suite="abel.tests.run_cli"`.